app/app.py

```diff
@@ -0,0 +1,32 @@
+import gspread
+from google.oauth2.service_account import Credentials
+from dotenv import load_dotenv
+import json
+import os
+import time
+import redis
+# Define the scope of the application
+scope = ["https://www.googleapis.com/auth/spreadsheets"]
+
+load_dotenv()
+credentials_json = os.getenv("SECRET")
+# Parse the credentials from JSON string to a dictionary
+credentials_dict = json.loads(credentials_json)
+# Use the credentials dictionary to authorize
+credentials = Credentials.from_service_account_info(credentials_dict)
+client = gspread.authorize(credentials)
+
+# Open the spreadsheet by name or by sheet's URL
+sheet = client.open("Copy of Copy of Grades and Student Dashboard").get_worksheet(4)
```

Comment:
Ideally throw this in a config file, but worst case declare it as a global constant.